task_1/html_task/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>

```

Comment:
Why different files? Why not have two different commits to make it easier?

Reply:
Made two separate commits the second